blog/02-02-2023-connect-and-interact-with-celo-using-web3onboard-library/connect-and-interact-with-celo-using-web3onboard-library.md

```diff
@@ -0,0 +1,1106 @@
+        height: '120px',
+        border: '0.1em solid purple',
+        color: green[400],
+        borderRadius: '6px',
+        textAnchor: 'start',
+      }} 
+      variant="text" onClick={async() => await connect() }
+      className='connectButton'
+    >
+      <Typography variant={"h6"} >Connect Wallet</Typography>
+    </Button>
+  );
+}
+
+export default ConnectButton;
+
+```
+
+## components/LandingPage.tsx
+
+Import the ConnectButton into the `LandingPage.tsx` and the new code should look thus:
+
+```bash
+import React, { useState} from "react";
+import Typography from "@mui/material/Typography";
+import Container from "@mui/material/Container";
+import { PageProps } from "@/interfaces";
+import { green, purple } from "@mui/material/colors";
+import Footer from "./Footer";
+import Link from "@mui/material/Link";
+import Stack from "@mui/material/Stack";
+import ConnectButton from "./ConnectButton";
+
+export default function LandingPage(props: PageProps) {
+  const { handleConnect } = props;
+
+  return (
+    <React.Fragment>
+      <Container maxWidth='md'>
+        <div className="topDiv">
+          <Stack className="divHeader" textAlign={"revert-layer"} >
+            <Typography component={"main"} variant="h2">Celo StakeVerse V2</Typography>
+            <Typography component={"main"} variant="h3">Stake Celo to earn RTK Token</Typography>
+            <Typography component={"main"} variant="h5">Made by <span>
+              <Link color="inherit" href="https://github.com/bobeu" style={{color: green[600]}}>
+                Isaac Jesse 
+              </Link> a.k.a Bobelr | Bobeu</span>
+            </Typography>
+            <Link color="inherit" href="https://github.com/bobeu/stakingdapp-on-celo" style={{color: purple[700]}}>
+              Source code
+            </Link>
+          </Stack>
+          <div className="divHeader"></div>
+          <div className="lowerDiv">
+            <ConnectButton connect={handleConnect} />
+          </div>
+        </div>
+      </Container>
+      <Footer sx={{ mt: 8, mb: 4 }} />
+    </React.Fragment>
+  );
+}
+
+```
+
+## components/App.tsx - 
+
+- Lastly, modify the `components/App.tsx` file to reflect our changes. The following are the important changes I have made to the file.
+
+- We call the 'useWallets' hook to get the currently connectedWallet.
+- `handleAccountChange` updated the account address when `stakeOnBehalf` is invoked. The function requires an address as argument hence 'handleAccountChange'.
+- Modify the `displayContractFunctions` function to display `stakeOnBehalf` contract function from the ABI.
+- We extract the provider from the current connected wallet.
+- Add a new `stake` function to avoid code redundancy in the switch statement.
+- Modify the first case in the switch statement.
+- Add a new case to the switch statement to handle `stakeOnBehalf` when it is called.
+- Add a new `TextField` component to take 'account' address from user.
+
+```bash
```

Comment:
Isn't this supposed to ```js?

Reply:
Please read the guidelines for review. This is the correct format.

Reply:
Please read the guidelines for review. This is the correct format.